SalesforceSDK/Universal/Analytics/AILTNPublisher.cs

```diff
@@ -67,12 +67,15 @@ public class AiltnPublisher : IAnalyticsPublisher
                     var eventToPublish = events.ElementAt(i);
                     if (eventToPublish != null)
                     {
-                        var trackingInfo = new JObject {{CODE, AILTN}};
+                        var trackingInfo = new JObject { { CODE, AILTN } };
                         var data = new JObject();
                         var schemaType = eventToPublish[InstrumentationEvent.SCHEMA_TYPE_KEY];
                         data.Add(InstrumentationEvent.SCHEMA_TYPE_KEY, schemaType);
-                        eventToPublish[InstrumentationEvent.SCHEMA_TYPE_KEY].Remove();
-                        data.Add(PAYLOAD, eventToPublish.ToString());
+                        var property =
+                            eventToPublish.Children<JProperty>()
+                                .FirstOrDefault(p => p.Name == InstrumentationEvent.SCHEMA_TYPE_KEY);
+                        property?.Remove();
+                        data.Add(PAYLOAD, eventToPublish.ToString().Replace("\r", "").Replace("\n", "").Replace(" ", ""));
```

Comment:
If `eventToPublish` is `InstrumentationEvent` object, shouldn't this be `eventToPublish.toJson()` instead of `eventToPublish.toString()`? That might be the issue here. Could you try it?